packages/shared/src/components/widgets/ToggleWeekStart.tsx

```diff
@@ -22,6 +23,7 @@ export const ToggleWeekStart = ({
     <Radio
       name="freeze-days"
       value={getDefaultStartOfWeek(streak.weekStart)}
+      disabled={isUpdatingConfig}
```

Comment:
While updating is not good to have the field available, maybe we can have race between different calls and this can cause issues.